.github/workflows/packages.yml

```
@@ -1,11 +1,24 @@
 name: Build Mergin Plugin Packages
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      MERGIN_CLIENT_VER:
+        description: 'Mergin client version either a tag, release, or branch'
+        required: true
+        default: 'master'
+        type: string
+      GEODIFF_VER:
+        description: 'Geodiff version released on PyPI repository'
+        required: true
+        default: '2.0.4'
+        type: string
 env:
-  MERGIN_CLIENT_VER: "0.9.3"
-  GEODIFF_VER: "2.0.4"
+  # Assign the version provided by 'workflow_dispatch' if available; otherwise, use the default.
+  MERGIN_CLIENT_VER: ${{ inputs.MERGIN_CLIENT_VER != '' && inputs.MERGIN_CLIENT_VER|| '0.9.3' }}
+  GEODIFF_VER:  ${{ inputs.GEODIFF_VER != '' && inputs.GEODIFF_VER|| '2.0.4' }}
```

I know bash can be picky when it comes to spaces, not sure if this matters or not
```suggestion
  GEODIFF_VER:  ${{ inputs.GEODIFF_VER != '' && inputs.GEODIFF_VER || '2.0.4' }}
```

---

Added the comment:  
```
 # Assign the version provided by 'workflow_dispatch' if available; otherwise, use the default.
```

---

Added the space between operator just in case ;-)